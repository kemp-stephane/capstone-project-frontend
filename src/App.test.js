import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import { render } from '@testing-library/react'
import Home from './pages/Home'

it("renders app without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

 
it('renders homepage', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})