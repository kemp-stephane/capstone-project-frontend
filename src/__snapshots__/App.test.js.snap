// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage 1`] = `
<div>
  <div>
    <div
      class="MuiBox-root css-1dhj0oh"
    />
  </div>
</div>
`;
